.github/Makefile

```diff
@@ -1,3 +1,5 @@
+.SILENT:
```

Comment:
This will not be created but I used my branch to run the automation from, so changes didn't exist in the target branch (main) yet.